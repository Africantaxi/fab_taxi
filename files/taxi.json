{
  "data": [
    {
      "ads": {
        "insee": "999999",
        "numero": "string"
      },
      "driver": {
        "departement": "999",
        "professional_licence": "testing_licence"
      },
      "status": "free",
      "vehicle": {
        "licence_plate": "testing_uwsgi"
      }
    }
  ]
}
