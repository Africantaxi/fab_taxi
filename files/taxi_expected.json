{
    "data": [
        {
            "ads": {
                "insee": "999999", 
                "numero": "string"
            }, 
            "crowfly_distance": null, 
            "driver": {
                "departement": "999", 
                "professional_licence": "testing_licence"
            }, 
            "last_update": null, 
            "operator": null, 
            "position": {
                "lat": null, 
                "lon": null
            }, 
            "rating": 4.5, 
            "status": "free", 
            "vehicle": {
                "characteristics": null, 
                "color": null, 
                "constructor": null, 
                "licence_plate": "testing_uwsgi", 
                "model": "string", 
                "nb_seats": null
            },
            "operator": null
        }
    ]
}
